packages/training/deploy/local/run.sh

```
@@ -150,11 +159,20 @@ if [[ "$INTERACTIVE" == "true" ]]; then
+        python3 python/scripts/run_training.py
+        --profile "$PROFILE"
+        --steps "$STEPS"
+        --min-agents-per-window "$MIN_AGENTS"
+    )
+    
+    # Add HF dataset if specified
+    if [[ -n "$HF_DATASET" ]]; then
+        TRAIN_CMD+=(--hf-dataset "$HF_DATASET")
+    fi
+    
+    "${DOCKER_CMD[@]}" "$IMAGE" "${TRAIN_CMD[@]}" $EXTRA_ARGS
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)